### ISSUE TITLE
RAML 1.0 support

### ISSUE THREAD

**Issue by lgelfan** [farolfo/raml-server#3]
Planning support for RAML 1.0? Is this still being maintained?

Thanks.

**Comment by farolfo** [comment]
Hello, yes it is maintained.
The raml-server makes usage of the raml-mocker project, so we first need to check all RAML 1.0 support is there and then making the changes here. Working on it, not with an ETA yet but I can keep you posted here. 
Is there anything in particular from RAML 1.0 you would like to see here? Thanks Iars !!

**Comment by lgelfan** [comment]
Hi @farolfo, thanks for the update. I just started a project that requires documentation and a mock server for an API and am trying both Swagger 2 and RAML 1.0 (via their API Workbench) - it just seem that 1.0 supports more data types and sample data, plus I already started the spec and am would like to compare the latest versions. So far I like RAML a bit better, just wonder about the support and adoption as the community seems a lot smaller.

**Comment by lufeihaidao** [comment]
Yes we need raml 1.0 support

**Comment by farolfo** [comment]
Thanks both for the interest in the project and in RAML ! RAML 1.0 integration is targeted for the next release which is soon. ;)

**Comment by supertylerc** [comment]
Any update on RAML 1.0 support?

**Comment by farolfo** [comment]
Hey ! I'm on vacations and offline most of the time so haven't been working on it lately. Next release would be targeted in a month with raml 1.0 support. This is a personal project so any PR or contribution will be very welcomed ;)
Thanks for the interest @supertylerc !

**Comment by jackreyneke** [comment]
Hi there Faralfo, thanks for the great project. Any update on RAML 1.0 support?

**Comment by neilhosey** [comment]
Brilliant tool, was there any update on when RAML 1.0 will be supported?  will be a good day when this happens..

**Comment by Marthyn** [comment]
I would like to have 1.0 too, also be willing to help you build it! Is there a branch already on which i can continue?

**Comment by neilhosey** [comment]
me too.. any help needed let me know!

**Comment by emanb29** [comment]
+1

**Comment by naterkane** [comment]
c'mon now. raml-mocker is also a dead project that this is relying on, it's never going to happen, unless people submit PRs and bully the project owners into letting the project move forward.
someone forked raml-mocker and created raml-1-mocker, which uses (a very stale version) of raml-1-parser. it's on npm, and there are no tests, and it does not work.

mulesoft doesn't give two shits about any of this, and the only non-dead project related to any of this is  https://github.com/raml-org/raml-js-parser-2

**Comment by farolfo** [comment]
Hi @naterkane , I don't see the the not-supported warning on the raml-mocker github site, so can't rely on that yet formally :/
This project (raml-server) it IS being maintained, but is just a 1-person project that I spend my free time and use it in my own projects, that's why I created it, for me, if anyone else can gain value with it then great, but I can't commit to any plan but github blocker issues and merging and testing PRs.
And of course I'm not talking from MuleSoft but from my own personal project, please forward to the raml-org any comment you have for them as this is just a personal project :)

**Comment by gunta** [comment]
+1

**Comment by napravicukod** [comment]
+1

**Comment by wis3guy** [comment]
@farolfo  Any update on this? I could really use a mock server like this ... for working with RAML 1.0 files.

**Comment by snowmac** [comment]
Any update on this?

**Comment by farolfo** [comment]
Yes guys sorry for the late reply. I'm working on the RAML 1.0 support on my free time so I'm not sure of an ETA yet, but working on it.
Any PR is welcome also 😃 
Thanks for the patience and the interest in the project :D

**Comment by snowmac** [comment]
Can you push what you have and we can contribute to it?

**Comment by yuranos** [comment]
Guys, is it gonna be released anywhere soon?

**Comment by farolfo** [comment]
Guys, sorry for the delay on the response. 
To be honest, the only one working on this is myself, and the project is NOT actively maintained, as for what I use it it fills my needs.

Any PR is welcome though, and please sorry for any inconvenience.
Thanks for understanding :)
